www/src/data/sidebars/doc-links.yaml

```diff
@@ -191,6 +191,13 @@
       items:
         - title: Adding search
           link: /docs/adding-search/
+          items:
+            - title: "Adding search with Algolia*"
```

Comment:
The apostrophes `"` aren't used anywhere else, so you should probably don't use them for coherence

Reply:
```suggestion
            - title: Adding search with Algolia*
```

Reply:
Thanks, I could have sworn I copied the quote style from elsewhere in the document, but clearly I was just making things up 🤔